tests/FakeItEasy.Specs/FakeItEasy.Specs.csproj

```diff
@@ -11,7 +11,11 @@
   </PropertyGroup>
 
   <PropertyGroup Condition="'$(TargetFramework)' == 'net461'">
-    <DefineConstants>$(DefineConstants)</DefineConstants>
+    <DefineConstants>$(DefineConstants);FEATURE_IN_PARAM</DefineConstants>
+  </PropertyGroup>
+
+  <PropertyGroup>
+    <LangVersion>latest</LangVersion>

```

Comment:
If you prefer, could go to 7.2.

Reply:
`latest` is fine, but maybe it should be set in the root Directory.Build.props, in order to affect all projects.